### PR TITLE
course details info (desc, price, duration in minutes)

### PR DESCRIPTION
**Pull request by marcinbutora**

### CODE REVIEW

File: training-service/src/app/content/course/component/course-list/course-list.component.ts [hunk 1]
```diff
@@ -10,6 +10,10 @@ import { CourseService } from '../../service/course.service';
   styleUrls: ['./course-list.component.scss']
 })
 export class CourseListComponent implements OnInit {
+
+  total!: 0;
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.ts]: please rename to total, 
remove !

File: training-service/src/app/content/course/component/course-list/course-list.component.html [hunk 2]
```diff
@@ -5,6 +5,10 @@
         <input type="checkbox">
           <span>{{ item.name }}</span>
       </label>
+      <span>
+        opis: {{ item.description }}, czas trwania: {{ item.duration * 1000 | date:'mm:ss' }} minut,
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.html]: please extract item.duration * 1000 to a function and do not do math operations in template.

**Review by holistic-fox — CHANGES_REQUESTED** [review]

File: training-service/src/app/content/course/component/course-list/course-list.component.html (hunk 2 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.html]: done

File: training-service/src/app/content/course/component/course-list/course-list.component.ts (hunk 1 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.ts]: done

File: training-service/src/app/content/course/component/course-list/course-list.component.ts [hunk 3]
```diff
@@ -19,12 +23,14 @@ export class CourseListComponent implements OnInit {
...
-  course: Course[] = [];
-
   constructor(private courseService: CourseService) { }
 
-  ngOnInit(): void{
+  ngOnInit() {
     this.course = this.courseService.getCourseList();
   }
 
+  convertDurationToTime(item: number):any {
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.ts]: please add proper return type and convert to arrow function

**Review by holistic-fox — CHANGES_REQUESTED** [review]

File: training-service/src/app/content/course/component/course-list/course-list.component.ts (hunk 3 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.ts]: done

**Review by holistic-fox — APPROVED** [review]

**Pull request merged**